### PR TITLE
sci-biology/velvet: expose additional variables as described in manual and needed at compile time

### PR DESCRIPTION
**Pull request by mmokrejs**

### CODE REVIEW

File: sci-biology/velvet/velvet-1.2.10.ebuild [hunk 1]
```diff
@@ -51,8 +53,11 @@ src_prepare() {
 		CFLAGS="${CFLAGS}"
 		OPT="${CFLAGS}"
 	)
+	if use openmp; then MAKE_XOPTS+=( OPENMP=1 ); fi
```
Comment (jlec) [inline sci-biology/velvet/velvet-1.2.10.ebuild]: Just shorten this to ```use openmp && MAKE_XOPTS+=( OPENMP=1 )```. Actually, do this for the rest as well.

File: sci-biology/velvet/velvet-1.2.10.ebuild [hunk 2]
```diff
@@ -53,11 +53,11 @@ src_prepare() {
 		CFLAGS="${CFLAGS}"
 		OPT="${CFLAGS}"
 	)
-	if use openmp; then MAKE_XOPTS+=( OPENMP=1 ); fi
-	if [[ ${VELVET_MAXKMERLENGTH} != "" ]]; then MAKE_XOPTS+=( MAXKMERLENGTH=${VELVET_MAXKMERLENGTH} ); fi
-	if [[ ${VELVET_CATEGORIES} != "" ]]; then MAKE_XOPTS+=( CATEGORIES=${VELVET_CATEGORIES} ); fi
-	if [[ ${VELVET_BIGASSEMBLY} != "" ]]; then MAKE_XOPTS+=( BIGASSEMBLY=${VELVET_BIGASSEMBLY} ); fi
-	if [[ ${VELVET_LONGSEQUENCES} != "" ]]; then MAKE_XOPTS+=( LONGSEQUENCES=${VELVET_LONGSEQUENCES} ); fi
+	use openmp && MAKE_XOPTS+=( OPENMP=1 )
+	test "${VELVET_MAXKMERLENGTH}" != "" && MAKE_XOPTS+=( MAXKMERLENGTH=${VELVET_MAXKMERLENGTH} )
```
Comment (jlec) [inline sci-biology/velvet/velvet-1.2.10.ebuild]: Better use
```[[ -z "${VELVET_MAXKMERLENGTH}" ]] || MAKE_XOPTS+=( MAXKMERLENGTH=${VELVET_MAXKMERLENGTH} )```

File: sci-biology/velvet/velvet-1.2.10.ebuild [hunk 3]
```diff
@@ -26,11 +26,13 @@ src_prepare() {
...
-	elog "To adjust the MAXKMERLENGTH or CATEGORIES parameters as described in the manual,"
-	elog "please set the variables VELVET_MAXKMERLENGTH or VELVET_CATEGORIES in your"
-	elog "environment or /etc/make.conf, then re-emerge the package. For example:"
+	elog "To adjust the MAXKMERLENGTH, CATEGORIES, BIGASSEMBLY, LONGSEQUENCES parameters"
+	elog "as described in the PDF manual, please set the variables by prepending VELVET_ in"
+	elog "front of it. For example VELVET_MAXKMERLENGTH, VELVET_CATEGORIES, ..."
+	elog "Set them either in your environment or in /etc/portage/make.conf, then re-emerge"
+	elog "the package. For example:"
 	elog "  VELVET_MAXKMERLENGTH=NN emerge [options] velvet"
 	MAKEOPTS+=" -j1"
```
Comment (SoapGentoo) [inline sci-biology/velvet/velvet-1.2.10.ebuild]: This is bad (but common style in Gentoo): you shouldn't ever muck with `MAKEOPTS`, instead appending `-j1` to `emake` calls.
